en/02_Developer_Guides/15_Customising_the_Admin_Interface/06_Javascript_Development.md

```diff
@@ -567,59 +412,22 @@ $('.LeftAndMain').entwine('ss', function($){
      */
     publicMethod: function(data, status) {
       return '<ul>'
-        + /...
+        + //...
         + '</ul>';
     },
 
     /**
-
      * Won't show in documentation, but still worth documenting.
      *
      * @return {String} Something else.
      */
     _privateMethod: function() {
       // ...
     }
-  };
-]]);
-```
-
-### Unit Testing
-
-It is important to verify that your code actually does what it says, and the best way to ensure this are **automated
-tests**. For jQuery, we use two different tools with different uses: **unit testing** with
-[QUnit](http://docs.jquery.com/QUnit) (also used by the jQuery team for the core libraries), and **behaviour driven
-testing** with [JSpec](http://visionmedia.github.com/jspec/). There are overlaps between the two solutions, if in doubt
-start with JSpec, as it provides a much more powerful testing framework.
-
```

Comment:
This information was just flat out wrong. Easier to just remove it - people know to test their stuff.